### DISCUSSION TITLE
how to change column if it's a fk?

### DISCUSSION THREAD

**Discussion by imiskolee** [directus/directus#13607] [Feature Requests]
Hi,team:

We know MySQL can not allow change columns if it's an FK, I am asking how to let it work.

some solutions from stackoverflow:

https://stackoverflow.com/questions/13606469/cannot-change-column-used-in-a-foreign-key-constraint

maybe remove and create again can work on directus.

**Comment by rijkvanzanten** [comment]
What change were you applying that didn't work in Directus currently?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
